index.js

```diff
@@ -79,7 +79,8 @@ const checkVersion = (engineName, command) => {
   let localVersionValid = null;
 
   try {
-    const localBinFolder = execSync('npm bin').toString().trim();
+    const localRootFolder = execSync('npm root').toString().trim();
```

Comment:
The previous output would have been like:
```
/Users/dominicfraser/Documents/repos/my-repo/packages/common/node_modules/.bin'
```

The new output would be like

```
/Users/dominicfraser/Documents/repos/my-repo/packages/common/node_modules'
```

We take this then prepend `/.bin` below.

Reply:
This has been tested on NPM 8 and NPM 9, on local machine and in our CI.